bench-tests/bt-0000-0001/GlusterBench.py

```diff
@@ -183,7 +183,7 @@ def make_report(size, average_seq_write, average_seq_read, average_rand_write, a
     print "Smallfile ls -l   " + str(size) + "   file size: " + str(average_smallfile_ls)
 
 def run_command(command_in):
-    p = subprocess.Popen(command_in, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    p = subprocess.Popen(command_in, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, shell=True)
```

Comment:
I think we should edit these to be proper dictionaries rather than passing `shell=True`.